### DISCUSSION TITLE
WISE on Malcolm

### DISCUSSION THREAD

**Discussion by KchChr** [cisagov/Malcolm#345] [Q&A] [answered]
How can i add enrichement with WISE (Arkime) to malcolm?
I want a query to an external opensearch for a value and a boolean field in my malcolm-opensearch which is true, if there is a hit in the external opensearch, false otherwise.

**Comment by mmguero** [comment, marked as answer]
A Malcolm enhancement for exposing Arkime's WISE enrichment to user is currently in progress by developer @piercema, you can track it the issue tracker [right here](https://github.com/cisagov/Malcolm/issues/377).

**Comment by mmguero** [comment]
Closing this discussion as duplicate of #377
